README.md

```diff
@@ -63,6 +63,19 @@ urlpatterns = [
     )
 ```
 
+#### Topic pages
+- Topic pages are optional as well and can be enabled by using the view `canonicalwebteam.blog.django.views.topic`. The view takes the topic slug to fetch data for and a template path to load the correct template from.
+
+**urls.py**
+```python
+path(
+		r"blog/topic/kubernetes",
```

Comment:
Oh just one tiny point - could you make this `/topics`?

Reply:
will do